notebooks/official/training/hyperparameter_tuning_tensorflow.ipynb

```diff
@@ -0,0 +1,1225 @@
+        "metric_spec = {\"accuracy\": \"maximize\"}"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "d3acfccaa053"
+      },
+      "source": [
+        "Once the specs are defined, you'll create a `CustomJob`, which is the common spec that will be used to run your job on each of the hyperparameter tuning trials."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "2bc47496bcfd"
+      },
+      "outputs": [],
+      "source": [
+        "my_custom_job = aiplatform.CustomJob(\n",
+        "    display_name=\"horses-humans-sdk-job\",\n",
+        "    worker_pool_specs=worker_pool_specs,\n",
+        "    staging_bucket=BUCKET_URI,\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "c82fd780f019"
+      },
+      "source": [
+        "Then, create and run the HyperparameterTuningJob."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "d7a1878e7872"
+      },
+      "outputs": [],
+      "source": [
+        "hp_job = aiplatform.HyperparameterTuningJob(\n",
+        "    display_name=\"horses-humans-sdk-job\",\n",
+        "    custom_job=my_custom_job,\n",
+        "    metric_spec=metric_spec,\n",
+        "    parameter_spec=parameter_spec,\n",
+        "    max_trial_count=5,\n",
+        "    parallel_trial_count=3,\n",
+        ")\n",
+        "\n",
+        "hp_job.run()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "956dc68681da"
+      },
+      "source": [
+        "There are a few arguments to note:\n",
+        "\n",
+        "\n",
+        "* **max_trial_count**: You'll need to put an upper bound on the number of trials the service will run. More trials generally leads to better results, but there will be a point of diminishing returns, after which additional trials have little or no effect on the metric you're trying to optimize. It is a best practice to start with a smaller number of trials and get a sense of how impactful your chosen hyperparameters are before scaling up.\n",
+        "\n",
+        "* **parallel_trial_count**: If you use parallel trials, the service provisions multiple training processing clusters. Increasing the number of parallel trials reduces the amount of time the hyperparameter tuning job takes to run; however, it can reduce the effectiveness of the job overall. This is because the default tuning strategy uses results of previous trials to inform the assignment of values in subsequent trials.\n",
+        "\n",
+        "* **search_algorithm**: You can set the search algorithm to grid, random, or default (None). The default option applies Bayesian optimization to search the space of possible hyperparameter values and is the recommended algorithm. You can learn more about this algorithm here."
```

Comment:
Is "none" the default or is "default" none? Or does "none" refer to leaving this configuration blank?